docs/components/README.md

```diff
@@ -1 +1,54 @@
 # Components for Community Building
+
+We at Rocket.Chat believe in  building susatinable online communities which can provide a holistic environment to all members around the world :earth_americas:. To enhance the community building experience we provide our our community builders with an array of Components which can help your communities grow to the moon :rocket:
```

Comment:
Please create a sub-directory named `leaderboard` and put your docs in the README.md there.

At this level there should just be a link from a list item `leaderboard` to that subdirectory.

This will allow the addition of other components to this page.

Reply:
Got it ! Will just do that.